static/app/components/events/groupingInfo/groupingInfo.tsx

```diff
@@ -99,7 +127,12 @@ const ConfigHeader = styled('div')`
   display: flex;
   justify-content: space-between;
   gap: ${space(1)};
```

Comment:
definitely not required but while you're in this file - might be worth replacing all the usages of `space` with `theme.space`